.travis.yml

```diff
@@ -4,14 +4,15 @@ language: node_js
 matrix:
   include:
 
-    # Run linter once
+    # Run linter only, just once
     - node_js: '6'
       env: LINT=true
 
     # Run tests
-    - node_js: '4'
-    - node_js: '5'
+    - node_js: 'node'
```

Comment:
Hi :) I'd be tempted to be explicit about this and specify `7`, then add `8` when it's released – more control. But then this is a Springer Nature repo and not Pa11y, so :man_shrugging: